idc_index/index.py

```diff
@@ -469,14 +469,15 @@ def get_viewer_URL(
         return viewer_url
 
     def _validate_update_manifest_and_get_download_size(
-        self, manifestFile, downloadDir, validate_manifest
+        self, manifestFile, downloadDir, validate_manifest=False
```

Comment:
this is an internal function that's called by download_from_manifest. So the validate_manifest is controlled at download_from_manifest. So this change must be reverted.